sdk/resource/process_test.go

```diff
@@ -118,7 +118,11 @@ func TestCommandArgs(t *testing.T) {
 }
 
 func TestRuntimeName(t *testing.T) {
-	require.EqualValues(t, runtime.Compiler, resource.RuntimeName())
+	if runtime.Compiler == "gc" {
```

Comment:
I'm not super fond of this condition in a test. If someone has an alternative approach, I'm happy to hear it :)